Formula/rdiff-backup.rb

```diff
@@ -21,7 +21,7 @@ class RdiffBackup < Formula
 
   def install
     python3 = "python3.11"
-    system python3, *Language::Python.setup_install_args(prefix, python3), "--install-data=#{prefix}"
+    system python3, "-m", "pip", "install", "--prefix=#{prefix}", "--no-deps", "--no-build-isolation", "."
```

Comment:
Do we maybe want to add `std_pip_install_args` to `brew` for this?

Reply:
yup agreed, I'll add that